habitat-lab/habitat/datasets/rearrange/samplers/object_sampler.py

```diff
@@ -344,24 +350,23 @@ def _is_accessible(
     ) -> bool:
         """
         Return if the object is within a threshold distance of the nearest
-        navigable point and that the nearest navigable point is on the same
-        navigation mesh.
+        navigable point, in which the nearest navigable point is on the same
+        navigation mesh of the object.
 
         Note that this might not catch all edge cases since the distance is
         based on Euclidean distance. The nearest navigable point may be
         separated from the object by an obstacle.
         """
         if self.nav_to_min_distance == -1:
             return True
-        snapped = sim.pathfinder.snap_point(obj.translation)
-        island_radius: float = sim.pathfinder.island_radius(snapped)
+
+        snapped = sim.pathfinder.snap_point(
```

Comment:
Minor note: this will be NaN if the snap fails. The distance checking logic will return False with comparison to NaN, so it works out.

Reply:
Yes, I also noticed this. thank you, alex!